.github/workflows/cron-storage-limits.yaml

```diff
@@ -0,0 +1,41 @@
+name: Storage Limit Email Notifications
+
+on:
+  schedule:
+    - cron: '0 5 * * *'
```

Comment:
just to check, this means run at 5am every day. Is the plan to just run this once or day or is it something we should run more regularly? Will it email a user that is at their limit every time it is run? Would a user that hits their limit at 5:01am only be informed 24hrs later?

https://crontab.guru/#0_5_*_*_*

Reply:
> just to check, this means run at 5am every day. 
Yep, kind of randomly chosen with the vague idea of not running _all_ our cron jobs at midnight and taking the platform down!

> Will it email a user that is at their limit every time it is run?
It will only email them once per day (or once per week, depending on the email type), so we could run the cron more often than this if we want to. Email frequency does not increase with cron frequency!

> Would a user that hits their limit at 5:01am only be informed 24hrs later?
Yes, and I think you raise a good point here. I don't really know what the typical pattern is for how (fast) a user tends to reach their limit, and therefore whether this is likely to be significant..?

> Is the plan to just run this once or day or is it something we should run more regularly?
Well, given the answers above, I think that comes down to what balance we want to strike between notifying users as quickly as possible vs. the potential strain it puts on the system (especially as we scale to more users). And maybe the unlikely possibility that there's a bug which causes us to spam everyone, so if we're feeling timid then we might want to deploy it set to once per day to start with and then crank it up. But given that we've got fairly good test coverage and we'll test it on staging before we stick it on prod, I'm not feeling timid!

How about every 6 hours?

Reply:
Moving this conversation in the main first iteration PR #851 in order to have all the sub PRs merged and tidied up in one place.